src/api/middleware/application.js

```diff
@@ -0,0 +1,13 @@
+import CompanyApplication, { CompanyApplicationRules } from "../../models/CompanyApplication.js";
+import { APIError, ErrorTypes } from "./errorHandler.js";
+import { StatusCodes as HTTPStatus } from "http-status-codes/build/cjs/status-codes.js";
+import { VALIDATION_LINK_EXPIRATION } from "../../models/constants/ApplicationStatus.js";
+import { SECOND_IN_MS } from "../../models/constants/TimeConstants.js";
+
+export const exceededCreationTimeLimit = async (req, res, next) => {
```

Comment:
I'm not sure about this function name, because it fails if an application exists, not when is exceeded. So I would call it something like `applicationDoesNotExist` or something similar. What do you think?

Reply:
The function fails when the application exists and is exceeded.